tests/core/test_kontainer_drivers.py

```diff
@@ -1,10 +1,15 @@
-import sys
-
 import pytest
+import sys
 
 from rancher import ApiError
 from .conftest import wait_for_condition, wait_until
 
+# FIXME update this to an actual release of kontainer engine example
+#  driver
+NEW_DRIVER_URL = "https://github.com/nathan-jenan-rancher/kontainer-" \
+                 "engine-example-driver/releases/download/v2.2.0/" \
```

Comment:
wrong version. you want `v0.2.2` not `2.2.0`

Reply:
@nathan-jenan-rancher i did the release https://github.com/rancher/kontainer-engine-driver-example/releases/tag/v0.2.1

Reply:
I will update the url.